src/DssSpellCollateral.sol

```diff
@@ -0,0 +1,179 @@
+        // ChainlogAbstract(CHAINLOG).setAddress("<join-name>", <join-address>);
+        // ChainlogAbstract(CHAINLOG).setAddress("<clip-name>", <clip-address>);
+        // ChainlogAbstract(CHAINLOG).setVersion("<new-version>");
+    }
+
+    function offboardCollaterals() internal {
+
+        address spotter = DssExecLib.spotter();
+
+        // Offboard UNI-A
+        // https://vote.makerdao.com/polling/QmSfLS6V#poll-detail
+        // https://forum.makerdao.com/t/signal-request-offboard-uni-univ2daieth-univ2wbtceth-univ2unieth-and-univ2wbtcdai/15160
+
+        DssExecLib.setIlkLiquidationPenalty("UNI-A", 0);
+        DssExecLib.setKeeperIncentiveFlatRate("UNI-A", 0);
+        DssExecLib.linearInterpolation({
+            _name:      "UNI-A Offboarding",
+            _target:    spotter,
+            _ilk:       "UNI-A",
+            _what:      "mat",
+            _startTime: block.timestamp,
+            _start:     CURRENT_UNI_A_MAT,
+            _end:       TARGET_UNI_A_MAT,
+            _duration:  30 days
+        });
+
+        // Offboard UNIV2DAIETH-A
+        // https://vote.makerdao.com/polling/QmQUozNn#poll-detail
+        // https://forum.makerdao.com/t/signal-request-offboard-uni-univ2daieth-univ2wbtceth-univ2unieth-and-univ2wbtcdai/15160
+
+        DssExecLib.setIlkLiquidationPenalty("UNIV2DAIETH-A", 0);
+        DssExecLib.setKeeperIncentiveFlatRate("UNIV2DAIETH-A", 0);
+        DssExecLib.linearInterpolation({
+            _name:      "UNIV2DAIETH-A Offboarding",
+            _target:    spotter,
+            _ilk:       "UNIV2DAIETH-A",
+            _what:      "mat",
+            _startTime: block.timestamp,
+            _start:     CURRENT_UNIV2DAIETH_A_MAT,
+            _end:       TARGET_UNIV2DAIETH_A_MAT,
+            _duration:  30 days
+        });
+
+        // Offboard UNIV2WBTCETH-A
+        // https://vote.makerdao.com/polling/QmY3YsDB#poll-detail
+        // https://forum.makerdao.com/t/signal-request-offboard-uni-univ2daieth-univ2wbtceth-univ2unieth-and-univ2wbtcdai/15160
+
+        DssExecLib.setIlkLiquidationPenalty("UNIV2WBTCETH-A", 0);
+        DssExecLib.setKeeperIncentiveFlatRate("UNIV2WBTCETH-A", 0);
+        DssExecLib.linearInterpolation({
+            _name:      "UNIV2WBTCETH-A Offboarding",
+            _target:    spotter,
+            _ilk:       "UNIV2WBTCETH-A",
+            _what:      "mat",
+            _startTime: block.timestamp,
+            _start:     CURRENT_UNIV2WBTCETH_A_MAT,
+            _end:       TARGET_UNIV2WBTCETH_A_MAT,
+            _duration:  30 days
+        });
+
+        // Offboard UNIV2UNIETH-A
+        // https://vote.makerdao.com/polling/QmUeYVa2#poll-detail
+        // https://forum.makerdao.com/t/signal-request-offboard-uni-univ2daieth-univ2wbtceth-univ2unieth-and-univ2wbtcdai/15160
+
+        DssExecLib.setIlkLiquidationPenalty("UNIV2UNIETH-A", 0);
+        DssExecLib.setKeeperIncentiveFlatRate("UNIV2UNIETH-A", 0);
```

Comment:
So was a little bit concerned that setting mat to 0 would cause pokes in the spotter to fail for this collateral (divide by 0).
I think it is fine since the megapoker does a low level call (so would not revert even if we keep running an old version).
Let me know if anyone sees a problem here.

Reply:
Where do we set `mat` to `0`?  We lerp the `mat` up from its start to the target, but we're only setting `chop` and `tip` to `0` here, which is fine.  `chop` is a little strange as an explicit 0 would produce a divide-by-zero in the dog, but the `DssExecLib` does the right thing and adds `WAD` to this value.

Reply:
ah right right, conused `mat` and `chop` for a minute.